package.json

```diff
@@ -7,32 +7,41 @@
     "prestart": "npm run build",
     "start": "node .",
     "dev": "tsnd --respawn src/app.ts",
-    "lint": "eslint . --ext .ts"
+    "lint": "eslint . --ext .ts",
+    "test": "mocha -r ts-node/register 'src/**/*spec.ts' --exit",
+    "coverage": "nyc npm run test -t ./.nyc-output report --reporter=html"
```

Comment:
Please add commands: 
- docker - run project in docker container
- docker:build - build container
- docker:test - run tests

Nice to have: 
- docker:test:cov - run tests with coverage
- docker:debug - run in debug mode